src/test/java/com/auth0/client/mgmt/UsersEntityTest.java

```diff
@@ -574,6 +574,13 @@ public void shouldThrowOnLinkUserIdentityWithNullSecondaryId() throws Exception
         api.users().linkIdentity("1", null, "auth0", null);
     }
 
+    @Test
+    public void shouldThrowOnLinkUserIdentityWithNullSecondaryToken() throws Exception {
```

Comment:
Let's repeat this one for the user id, which also needs to be set.

Reply:
Added. Sorry for the miss. Good catch.